chart/templates/certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "loki-rule-operator.fullname" . }}
+  labels:
+    {{- include "loki-rule-operator.labels" . | nindent 4 }}
+spec:
+  dnsNames:
+  -  {{ include "loki-rule-operator.fullname" . }}.{{ .Release.Namespace }}.svc
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: selfsigned-issuer
```

Comment:
I think the issuer should be configurable as well. 
And maybe we should include a [selfsigned issuer](https://cert-manager.io/docs/configuration/selfsigned/) in this chart as well.

Reply:
How about using `cert-manager.io/inject-apiserver-ca: true` instead of handling a certificate ?
https://cert-manager.io/docs/concepts/ca-injector/#injecting-the-kubernetes-api-server-ca

Reply:
>  you will require access to the private key of the Kubernetes cluster CA and you will need to manually rotate the webhook certificate.

Might not be a good idea..